frontend/components/forms/image-uploader/component.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ChangeEvent, useState } from 'react';
+
+import { FieldValues } from 'react-hook-form';
+import { useIntl } from 'react-intl';
+
+import Image from 'next/image';
+
+import Button from 'components/button';
+import Loading from 'components/loading';
+
+import { ImageUploaderProps } from './types';
+
+export const ImageUploader = <FormValues extends FieldValues>({
+  preview,
+  register,
+  name,
+  id,
+  text,
+  handleChangeImage,
+  registerOptions,
+  ...rest
+}: ImageUploaderProps<FormValues>) => {
+  const { formatMessage } = useIntl();
+  const [imagePreview, setImagePreview] = useState<string>();
+
+  const handleUploadImage = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.currentTarget.files?.length) {
+      const file = e.currentTarget.files[0];
+      const src = URL.createObjectURL(file);
+      setImagePreview(src);
+      handleChangeImage();
+    }
+  };
+
+  return (
+    <div className="flex items-center justify-start">
+      {preview && (
+        <div className="mr-5">
+          <Image
+            src={imagePreview || '/images/avatar.svg'}
+            width={48}
+            height={48}
+            className="rounded-full"
+            alt={formatMessage({ defaultMessage: 'profile image', id: 'OX08fr' })}
```

Comment:
🟠

If this component is reusable (not just used for the profile image), this alternative text should either be more generic (for example “Preview”; you don't have to say “Preview image” because the screen reader will already tell the user it's an image) or come from a prop.